project/SparkBuild.scala

```diff
@@ -347,7 +347,7 @@ object Assembly {
         .getOrElse(SbtPomKeys.effectivePom.value.getProperties.get("hadoop.version").asInstanceOf[String])
     },
     jarName in assembly <<= (version, moduleName, hadoopVersion) map { (v, mName, hv) =>
-      if (mName.contains("streaming-kafka-assembly")) {
+      if (mName.contains("streaming-kafka-assembly"|"streaming-mqtt-assembly")) {
```

Comment:
Just a head's up. This will need to be fixed once again when Flume python API gets merged.